lib/SimpleSAML/Module.php

```diff
@@ -63,6 +107,172 @@ public static function isModuleEnabled($module)
     }
 
 
+    /**
+     * Handler for module requests.
+     *
+     * This controller receives requests for pages hosted by modules, and processes accordingly. Depending on the
+     * configuration and the actual request, it will run a PHP script and exit, or return a Response produced either
+     * by another controller or by a static file.
+     *
+     * @param Request|null $request The request to process. Defaults to the current one.
+     *
+     * @return Response|BinaryFileResponse Returns a Response object that can be sent to the browser.
+     * @throws Error\BadRequest In case the request URI is malformed.
+     * @throws Error\NotFound In case the request URI is invalid or the resource it points to cannot be found.
+     */
+    public static function process(Request $request = null)
```

Comment:
Type-hinting objects is only supported on PHP7

Reply:
See here: http://php.net/manual/en/migration72.new-features.php

Reply:
Ah, my bad!